e2e-tests/web/browserstack/wdio-browserstack.conf.js

```diff
@@ -0,0 +1,72 @@
+const childProcess = require('child_process')
+const defaultConfig = require('../wdio.conf')
+const merge = require('deepmerge')
+
+const getGitBranch = () => {
+  return childProcess.execSync('git rev-parse --abbrev-ref HEAD').toString().trim()
+}
+
+const getGitHeadReference = () => {
+  return childProcess.execSync('git rev-parse --short HEAD').toString().trim()
+}
+
+const browserstackCaps = config => {
+  const prefix = 'IG DEV'
+  return {
+    'bstack:options': {
+      ...config,
+      buildName: `${prefix}: ${getGitBranch()}`,
+      sessionName: `${config.browserName.toLowerCase()}: ${getGitHeadReference()}`,
+      local: true,
+      debug: true,
+      projectName: 'integreat-app-web'
+    },
+    browserName: config.browserName,
+    acceptInsecureCerts: true
+  }
+}
+
+exports.config = merge(defaultConfig.config, {
+  maxInstances: 1,
```

Comment:
Can we set this to 2 or even 5? Did you try this? 

Asking because of native e2e tests

Reply:
yes. Just for the web e2e tests I don't think it makes sense as we will not use browserstack in the ci